health/health.d/cockroachdb.conf

```diff
@@ -41,7 +45,10 @@ component: CockroachDB
     every: 10s
      warn: $this > 0
     delay: down 15m multiplier 1.5 max 1h
-     info: number of ranges with fewer live replicas than needed for quorum
+     info: Number of unavailable ranges. \
+           There are ranges with fewer live replicas than needed for quorum. \
+           If a majority of a range replicas are on nodes that are unavailable, \
```

Comment:
Won't work because we replace quotes with spaces (`range's` => `range s`)

https://github.com/netdata/netdata/blob/f455b15d29f49a8a9869ea562f53d7da8c4a21b7/health/health_config.c#L1075-L1076

https://github.com/netdata/netdata/blob/f455b15d29f49a8a9869ea562f53d7da8c4a21b7/health/health_config.c#L499-L504

Reply:
I think it is acceptable w/o `'s`, but we can reword it if you have any suggestions.

Reply:
It’s technically not grammatically correct without the possessive. How, if at all, do we handle Unicode characters? If we just pass them through we should be able to just put a ’ there and have it work correctly.

Reply:
We can write it the following way
```suggestion
           If a majority of replicas of a range are on nodes that are unavailable, \
```